internal/exeutils/exeutils_windows.go

```diff
@@ -1,3 +1,25 @@
 package exeutils
 
+import (
+	"os"
+	"path/filepath"
+	"strings"
+
+	"github.com/ActiveState/cli/internal/fileutils"
+)
+
 const Extension = ".exe"
+
+// PathForExecutable returns the first path from the PATH env var for which the executable exists
+func PathForExecutable(executable string) string {
+	candidates := strings.Split(os.Getenv("PATH"), string(os.PathListSeparator))
+	exts := append([]string{""}, strings.Split(os.Getenv("PATHEXT"), string(os.PathListSeparator))...)
```

Comment:
In case the executable is eg., `python.exe` we do not want to append the PATHEXT extensions to it, as that would look for `python.exe.exe`.  I'll leave a comment.